config.yml

```
@@ -196,3 +196,63 @@ annotations-research:
       - "specimenID"
     assay:
       - "specimenID"
+
+veoibd:
+  parent: "syn25556537"
+  path_to_markdown: "inst/using-the-dccvalidator-veoibd.Rmd"
+  study_table: "syn25557724"
+  samples_table: "syn25557826"
+  annotations_table: "syn25558026"
+  annotations_link: "https://www.synapse.org/#!Synapse:syn20729790"
+  templates_link: "https://www.synapse.org/#!Synapse:syn20729790"
+  docs_tab:
+    include_tab: TRUE
+    include_upload_widget: FALSE
+    tab_name: "Study Documentation"
+    path_to_docs_markdown: "inst/study-documentation-pec.Rmd"
+  teams:
+    - "3382930"
```

```suggestion
    - "3382930"
    - "3425042"
```
This should probably have the admin team on it, as well. Also, I noticed that you, @cw-dvr8, are the only admin on that team. You may want to elevate Mette to admin.